orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -134,6 +134,11 @@ def __init__(
         # Assign self._torch_alloc and self._torch_free if self._use_external_gpu_allocator is True
         self._get_torch_gpu_allocator_function_addresses()
 
+        if self._runtime_options.enable_triton:
+            from onnxruntime.training.ort_triton import register_triton_op_executor
```

Comment:
I guess if this import fails, it means onnxruntime  was not compiled with the expected settings. Is it worth adding an error message telling that?

Reply:
If onnxruntime is not compiled with expected setting, the self._runtime_options.enable_triton is not possible to be true, the import here is not possible to be failed.